popup/main.js

```diff
@@ -2,26 +2,38 @@ let listEnvironments = document.querySelector(".list-environments")
 let btnAddEnv = document.getElementById("btn-add-env");
 
 let addEnvironment = function(url, color){
-	let environmentStr =`<input type="color" value="${ color || randomDefaultColor() }"><input type="text" placeholder="*.dev.example.com" value="${ url || ''}"><button class="btn-remove">Remove</button>`;
 	let environmentNode =  document.createElement("section");
 	environmentNode.classList.add("environment");
-	environmentNode.innerHTML = environmentStr;
-	environmentNode.querySelector(".btn-remove").addEventListener("click", removeEnvironment.bind(environmentNode), false);
-	environmentNode.querySelector('input[type="color"]').addEventListener("change", updateEnvironments, false);
-	environmentNode.querySelector("input").addEventListener("input", updateEnvironments, false);
+
+	let envColor = document.createElement("input");
+	envColor.type = "color";
+	envColor.value = color || randomDefaultColor();
+	envColor.addEventListener("change", updateEnvironments, false);
+	environmentNode.appendChild(envColor);
+	
+	let envDomain = document.createElement("input");
```

Comment:
You need to add 

`envDomain.type = "text";` 

Otherwise this line https://github.com/ylecuyer/envify/blob/master/popup/main.js#L30 doesn't work